v3/setup/splunk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# SPLUNK_FORWARD_SERVER shoudl be delcared as a config in the setup-credentials util
+
+source /etc/environment
+
+etcdctl set /splunk/SPLUNK_FORWARD_SERVER $SPLUNK_FORWARD_SERVER
```

Comment:
Sorry if that was confusing, to summarize:

* `secrets.json` file can do configs (set via etcd), but not environment variables currently (changing soon).
* For now, keep this file with line 7 below (`etcdctl set /images/splunkforwarder "adobeplatform/docker-splunk:latest"`) since that is a default.
* Remove line 6 and require it in `secrets.json` configs section instead.